src/helpers/accessibilityListeners.ts

```diff
@@ -0,0 +1,115 @@
+import { AccessibilityInfo } from 'react-native';
+
+import { TAccessibility } from '../types';
+
+// for detailed information: https://reactnative.dev/docs/accessibilityinfo#addeventlistener
+
+export const accessibilityListeners = (setAccessibility: (prev: any) => void) => {
+  /* Fires when the state of the bold text toggle changes. 
+  The argument to the event handler is a boolean. 
```

Comment:
i suggest not having the boolean explained here, a the method param already show it because of `ts`:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/1942953/224758449-c39ab99d-c692-4c74-b6ae-9b4395d4defc.png">